_data/accessibility-tests/search.yml

```diff
@@ -0,0 +1,165 @@
+title: Search
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey the search button's meaning.
+    summary_additional: When using the search button, you can read text to learn the button's meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Search button background and text meet color contrast requirements. 
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search button, the background and lettering are at least 4:5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: The width of the search button is at least 44 pixels.
+    summary_additional: |
+      When you select "inspect element" to review the button's size, you can see the total width of the button is at least 44 pixels.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.5
+  - summary: Search field and button views adjust on mobile.
+    summary_additional: |
+      When you rotate between portrait mode and landscape mode on a mobile device, the search button and field are not restricted to a single view.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.4
+  - summary: Magnifying glass icon is only used to indicate "search" in every instance on the site.
+    summary_additional: |
+      When you see or tab to the magnifying glass icon anywhere on the site, it will always and only be 
+      used to indicate search.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.4
+  - summary: The search feature appears in the same location on every page of the website.
+    summary_additional: |
+      When you navigate to different pages on the website, the search feature is always in the same place.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.3
+  - summary: Search instructions are visible and audible, are specific for intent, and persist.
+    summary_additional: |
+      When you use the search feature, you may hear instructions with a screen reader or see them on the page. 
+      Instructions may be specific depending on intent (e.g., search this site, search this directory). Instructions do not disappear.
```

Comment:
So maybe taking the last sentence and making it it's own check? I think that could work. Summary: Search instructions remain on screen. Additional: When using the search feature, the instructions do not disappear from view. Something along these lines?

Reply:
Would this be a second test with the same criterion number? I looked to see if there was a specific test I could tie to this but didn't find another that is suitable.

Reply:
Yes! I think it would be a second test for the same criteria. As the one test does seem a bit overwhelming. I think we have done that in the past with other test pages.

Reply:
I opted to use the word `persistent` since that was in the original summary.

>  **Search instructions are persistent.**
When using the search feature, the instructions remain on screen and do not disappear from view.


Updated in [418019e](https://github.com/uswds/uswds-site/pull/2694/commits/418019ed120e0f8fda8803d26f6f4b63d528b8bb)